easybuild/easyconfigs/u/UQTk/UQTk-3.1.0-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,63 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'CMakeMake'
+
+name = 'UQTk'
+version = '3.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.sandia.gov/UQToolkit/'
+description = """The UQ Toolkit (UQTk) is a collection of libraries and tools 
+for the quantification of uncertainty in numerical model predictions."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+# https://github.com/sandialabs/UQTk
+github_account = 'sandialabs'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_fix-blas-lapack.patch']
+checksums = [
+    '56ecd3d13bdd908d568e9560dc52cc0f66d7bdcdbe64ab2dd0147a7cf1734f97',  # v3.1.0.tar.gz
+    '3ef0e6a8efcedb60afd029b23302b33df51d027dbda93818004e057b27fa33f8',  # UQTk-3.1.0_fix-blas-lapack.patch
+]
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('SWIG', '4.0.1'),
+]
+
+dependencies = [
+    ('expat', '2.2.7'),
+    ('imkl', '2019.5.281'),
+    ('SUNDIALS', '5.1.0'),
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('matplotlib', '3.1.1', versionsuffix),
+    ('PyMC3', '3.8', versionsuffix),
+]
+
+configopts = "-DCMAKE_SUNDIALS_DIR=$EBROOTSUNDIALS -DPyUQTk=ON "
+buildopts = " && ctest"
+separate_build_dir = True
+
+modextrapaths = {'PYTHONPATH': ''}
```

Comment:
This should be:

```python
modextrapaths = {'PYTHONPATH': 'PyUQTk'}

Reply:
not true. I think those imports are from Python2 era - mixing absolute and relative and they just didn't used other Python scripts. Tests will pass if you can build the .so objects.

Reply:
OK